src/index.js

```diff
@@ -0,0 +1,276 @@
+  "#profile-description-input"
+);
+const editSubmitButton = editModal.querySelector(".modal__submit-btn");
+
+const deleteConfirmModal = document.querySelector("#confirm-delete");
+console.log(deleteConfirmModal);
+const deleteConfirmCloseBtn =
+  deleteConfirmModal.querySelector(".modal__close-btn");
+
+const CancelConfirmCloseBtn = deleteConfirmModal.querySelector(
+  ".modal__cancel-button"
+);
+
+const cardModal = document.querySelector("#add-card-modal");
+const cardForm = cardModal.querySelector(".modal__form");
+const cardModalCloseBtn = cardModal.querySelector(".modal__close-btn");
+const cardNameInput = cardModal.querySelector("#add-card-name-input");
+const cardLinkInput = cardModal.querySelector("#add-card-link-input");
+const cardSubmitButton = cardModal.querySelector(".modal__submit-btn");
+
+// Preview
+const previewModal = document.querySelector("#preview-modal");
+const previewModalImageEl = previewModal.querySelector(".modal__image");
+const previewModalCaptionEl = previewModal.querySelector(".modal__caption");
+const previewModalCloseButton = previewModal.querySelector(".modal__close-btn");
+
+// Card related elements
+const cardTemplate = document.querySelector("#card-template");
+const cardsList = document.querySelector(".cards__list");
+
+// Change avatar functionality
+const changeAvatarButton = document.querySelector(".profile__avatar-btn");
+const changeAvatarModal = document.querySelector("#edit-avatar-modal");
+const changeAvatarForm = changeAvatarModal.querySelector("#edit-avatar-form");
+const avatarSubmitButton =
+  changeAvatarModal.querySelector(".modal__submit-btn");
+const changeAvatarClose = changeAvatarModal.querySelector(".modal__close-btn");
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "d44cd1da-3eb5-4df7-b9ca-8d742a5da7d3",
+    "Content-Type": "application/json",
+  },
+});
+
+api
+  .getAppInfo()
+  .then(([cards, userInfo]) => {
+    profileName.textContent = userInfo.name;
+    profileDescription.textContent = userInfo.about;
+    const profileAvatar = document.querySelector(".profile__avatar");
+    profileAvatar.src = userInfo.avatar;
+
+    cards.forEach((item) => {
+      const cardEl = getCardElement(item);
+      cardsList.append(cardEl);
+    });
+  })
+  .catch(console.error);
+
+function openModal(modal) {
+  modal.classList.add("modal_opened");
+  addEscapeListener(); // Add
+}
+
+function closeModal(modal) {
+  modal.classList.remove("modal_opened");
+  removeEscapeListener(); // Remove
+}
+
+function handleEditFormSubmit(evt) {
+  evt.preventDefault();
+  editSubmitButton.textContent = "Saving...";
+  api
+    .editUserInfo({
+      name: editModalNameInput.value,
+      about: editModalDescriptionInput.value,
+    })
+    .then((data) => {
+      profileName.textContent = data.name;
+      profileDescription.textContent = data.about;
+      closeModal(editModal);
+      editSubmitButton.textContent = "Save";
```

Comment:
You need to **only** return the **default** button text in the `finally` block, so that the button could get the text regardless of whether the request succeeds or fails.